automatic/keepass-classic/keepass-classic.nuspec

```diff
@@ -1,29 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <title>KeePass Classic Password Safe</title>
+    <id>keepass-classic</id>
+    <version>1.32</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/keepass-classic</packageSourceUrl>
     <owners>chocolatey</owners>
-    <version>{{PackageVersion}}</version>
+    <title>KeePass Classic Password Safe</title>
     <authors>Dominik Reichl</authors>
+    <projectUrl>http://keepass.info/index.html</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/keepass.svg</iconUrl>
+    <copyright>Copyright © 2003-2017 Dominik Reichl</copyright>
+    <licenseUrl>http://keepass.info/help/v1/license.html</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>http://sourceforge.net/projects/keepass/files/KeePass%201.x/1.32/</projectSourceUrl>
```

Comment:
I'm not entirely sure if this is acceptable?
I've put a note about it in the description though.

/cc @ferventcoder @gep13